### ISSUE TITLE
Link error messages to text inputs using aria-describedby

### ISSUE THREAD

**Issue by sukhrajghuman** [AusDTO/dto-digitalmarketplace-frontend#794]
The error messages should be linked to the text inputs via the `aria-describedby` attribute. That way if a screen reader tabs to each field, they know what the error is.

See example 2 here:
https://www.w3.org/WAI/WCAG21/Techniques/aria/ARIA1
